hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientMultiMapProxy.java

```diff
@@ -148,6 +149,12 @@ public boolean remove(Object key, Object value) {
         MultiMapRemoveCodec.ResponseParameters resultParameters = MultiMapRemoveCodec.decodeResponse(response);
         return new UnmodifiableLazyList<V>(resultParameters.response, getSerializationService());
     }
+    public void delete(Object key) {
```

Comment:
Just a general style thing: We have a single newline between each function/block in Hazelcast.

Reply:
👍 fixed,   will do from now on.